.github/workflows/update.yml

```diff
@@ -0,0 +1,40 @@
+name: Update Data
+
+on:
+  schedule:
+  - cron: "0 0 8 * *"
+
+jobs:
+
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Set up Go
+      uses: actions/setup-go@v3
+      with:
+        go-version: 1.18
+        
+    - name: Get current date
+      id: date
+      run: echo "::set-output name=date::$(date +'%Y-%m-%d')"
+
+    - name: Update
+      run: make update
+
+    - name: Build
+      run: go build -v ./...
+
+    - name: Test
+      run: go test -v ./...
+      
+    - name: Create Pull Request
+      uses: peter-evans/create-pull-request@v4.0.3
+      branch: data/${{ steps.date.outputs.date }}
+      title: Data Update ${{ steps.date.outputs.date }}
+      body: |
+        Date updated on ${{ steps.date.outputs.date }} from https://bsb.auspaynet.com.au/
+      labels: data update
+      reviewers: timwmillard, luke-single
```

Comment:
@luke-single I added you on here but I can remove if you want?

Reply:
all good! looks great